### PR TITLE
add monitor & timeout parameters

### PR DESCRIPTION
**Pull request by lenhattan86**
Changes:
(1) In some cases, we don't need to monitor the process of creating/killing jobs. We can disable monitoring by indicating ```m=false``` 
(2) By default gorealis timeout is 20 seconds. It may need longer than that. We can change the timeout value using ```t=30```

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -68,6 +69,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&configFile, "config", "/etc/aurora/australis.yml", "Config file to use.")
 	rootCmd.PersistentFlags().BoolVar(&toJson, "toJSON", false, "Print output in JSON format.")
 	rootCmd.PersistentFlags().StringVarP(&logLevel, "logLevel", "l", "info", "Set logging level ["+internal.GetLoggingLevels()+"].")
+	rootCmd.PersistentFlags().Int64VarP(&timeout, "timeout", "t", 20, "Gorealis timeout (default: 20 seconds)")
```
Comment (ridv) [inline cmd/root.go:72]: Let's use [DurationVarP](https://godoc.org/github.com/spf13/pflag#DurationVarP) instead of Int64VarP

**Review by ridv — COMMENTED** [review]
One minor improvement, otherwise looks good.

File: cmd/root.go (hunk 1 above)
Comment (lenhattan86) [reply cmd/root.go:72]: will do.

**Review by ridv — APPROVED** [review]
LGTM thanks!

File: cmd/root.go [hunk 2]
```diff
@@ -68,6 +69,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&configFile, "config", "/etc/aurora/australis.yml", "Config file to use.")
 	rootCmd.PersistentFlags().BoolVar(&toJson, "toJSON", false, "Print output in JSON format.")
 	rootCmd.PersistentFlags().StringVarP(&logLevel, "logLevel", "l", "info", "Set logging level ["+internal.GetLoggingLevels()+"].")
+	rootCmd.PersistentFlags().DurationVarP(&timeout, "timeout", "t", 20, "Gorealis timeout (default: 20 seconds)")
```
Comment (misho-kr) [inline cmd/root.go:72]: This does not look right.

If someone enters "20s" the value that will be stored in `timeout` will be 20 x 10e9. Then on line 140 that will be multiplied by 10e9 (the value of `time.Second`) and the result will be certainly not be what you expect.

Comment (lenhattan86) [reply cmd/root.go:72]: good point will fix it.

Comment (ridv) [reply cmd/root.go:72]: good catch @misho-kr !

**Pull request merged**